environments/stg/backend_lb_config.yaml

```diff
@@ -102,6 +102,12 @@ gateways:
       - product: fpl
         component: case-service
 
+    # Family Private Law
```

Comment:
Please add change to bottom of file, this current PR will add to an Application Gateway that is currently full backend